core/server/services/xmlrpc.js

```diff
@@ -1,101 +0,0 @@
-var _ = require('lodash'),
-    xml = require('xml'),
-    config = require('../config'),
-    urlService = require('../services/url'),
-    common = require('../lib/common'),
-    request = require('../lib/request'),
-    settingsCache = require('./settings/cache'),
-
-    defaultPostSlugs = [
-        'welcome',
-        'the-editor',
-        'using-tags',
-        'managing-users',
-        'private-sites',
-        'advanced-markdown',
-        'themes'
-    ],
-    // ToDo: Make this configurable
-    pingList = [
-        {
-            url: 'blogsearch.google.com/ping/RPC2'
-        },
-        {
-            url: 'rpc.pingomatic.com'
-        }
-    ];
-
-function ping(post) {
-    var pingXML,
-        title = post.title,
-        url = urlService.utils.urlFor('post', {post: post}, true);
-
-    if (post.page || config.isPrivacyDisabled('useRpcPing') || settingsCache.get('is_private')) {
-        return;
-    }
-
-    // Don't ping for the default posts.
-    // This also handles the case where during ghost's first run
-    // models.init() inserts this post but permissions.init() hasn't
-    // (can't) run yet.
-    if (defaultPostSlugs.indexOf(post.slug) > -1) {
-        return;
-    }
-
-    // Build XML object.
-    pingXML = xml({
-        methodCall: [{
-            methodName: 'weblogUpdate.ping'
-        }, {
-            params: [{
-                param: [{
-                    value: [{
-                        string: title
-                    }]
-                }]
-            }, {
-                param: [{
-                    value: [{
-                        string: url
-                    }]
-                }]
-            }]
-        }]
-    }, {declaration: true});
-
-    // Ping each of the defined services.
-    _.each(pingList, function (pingHost) {
-        var options = {
-            body: pingXML,
-            timeout: 2 * 1000
-        };
-
-        request(pingHost.url, options)
-            .catch(function (err) {
-                common.logging.error(new common.errors.GhostError({
-                    err: err,
-                    message: err.message,
-                    context: common.i18n.t('errors.services.ping.requestFailed.error', {service: 'xmlrpc'}),
-                    help: common.i18n.t('errors.services.ping.requestFailed.help', {url: 'http://docs.ghost.org'})
```

Comment:
Are these error message used somewhere else?
If no, we can remove them :)

Reply:
There is a translation file in `server/translations/`. You can look up the full keys e.g. `errors.services.ping.requestFailed.help` by doing a project search. If they are not used, you can remove them from the translation file.

Reply:
slack.js file use:
https://github.com/TryGhost/Ghost/blob/a3091a30123bd08e274fe0c5c4537366f719cb74/core/server/services/slack.js#L34